independent-publisher-connectors/Cisco Webex List Meeting/apiDefinition.swagger.json

```diff
@@ -0,0 +1,197 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cisco Webex List Meetings",
+    "description": "The meeting API provides meeting details. The detail includes the client detail, meeting time, and meeting duration.",
+    "version": "1.0",
+    "contact": {
+      "name": "Woong Choi",
+      "url": "https://www.linkedin.com/in/woongchoi/"
+    }
+  },
+  "host": "webexapis.com",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/meetings": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@last72 please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.